<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MQTT Broker</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHost">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>164</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>broker.hivemq.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pingButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
   <property name="waiting_response" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_25">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>761</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>BME680</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEMPERATURA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HUMEDAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PRESSION</string>
    </property>
   </widget>
   <widget class="ManoMeter" name="pression_ind">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>69</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Shows the pressure</string>
    </property>
    <property name="whatsThis">
     <string>The bar meter widget displays the pressure attached to it</string>
    </property>
    <property name="maximum">
     <double>150000.000000000000000</double>
    </property>
    <property name="critical">
     <double>101300.000000000000000</double>
    </property>
    <property name="suffix">
     <string> [Pa]</string>
    </property>
    <property name="valueOffset">
     <double>-100.000000000000000</double>
    </property>
    <property name="digitOffset">
     <double>95.000000000000000</double>
    </property>
   </widget>
   <widget class="ThermoMeter" name="temp_ind">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>50</y>
      <width>81</width>
      <height>201</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Shows the pressure</string>
    </property>
    <property name="whatsThis">
     <string>The bar meter widget displays the pressure attached to it</string>
    </property>
    <property name="maximum">
     <double>70.000000000000000</double>
    </property>
    <property name="critical">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QwtThermo" name="humedad_ind">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>60</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="bmeslider">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>291</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bme">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="gas_ind">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Calidad del Aire (IAQ)</string>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="gas_status">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estado del Aire:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>326</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FRECUENCIA EJECUCIÃ“N</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="habilitar_alarma_bme">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Habilitar Alarma Temperatura</string>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="texto_alarma_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Estado Alarma Temperatura:</string>
    </property>
   </widget>
   <widget class="Led" name="led_alarma_bme">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Color Led component</string>
    </property>
    <property name="whatsThis">
     <string>Led indicator</string>
    </property>
    <property name="color">
     <color>
      <red>0</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </widget>
   <widget class="QwtKnob" name="umbral_alarma_bme">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="lowerBound">
     <double>1.000000000000000</double>
    </property>
    <property name="upperBound">
     <double>80.000000000000000</double>
    </property>
    <property name="scaleMaxMajor">
     <number>5</number>
    </property>
    <property name="scaleMaxMinor">
     <number>3</number>
    </property>
    <property name="scaleStepSize">
     <double>10.000000000000000</double>
    </property>
    <property name="totalSteps">
     <UInt>80</UInt>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="text_alarma_bme">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>...</string>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="texto_alarma_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Umbral Alarma Temperatura:</string>
    </property>
   </widget>
  </widget>
  <widget class="QwtTextLabel" name="board_status">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="plainText">
    <string>...</string>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="TextLabel_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>Estado del cliente de la placa:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Sensor Temp DS1621</string>
   </property>
   <widget class="QSlider" name="tempslider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_temp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:purple</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_temp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ÂºC</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>ADC</string>
   </property>
   <widget class="QPushButton" name="pushButton_adc">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:yellow</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="habilitar_alarma">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Habilitar Alarma ADC</string>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="texto_alarma">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Estado Alarma ADC:</string>
    </property>
   </widget>
   <widget class="Led" name="led_alarma">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Color Led component</string>
    </property>
    <property name="whatsThis">
     <string>Led indicator</string>
    </property>
    <property name="color">
     <color>
      <red>0</red>
      <green>255</green>
      <blue>0</blue>
     </color>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="texto_alarma_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Umbral Alarma ADC:</string>
    </property>
   </widget>
   <widget class="QwtKnob" name="umbral_alarma">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="lowerBound">
     <double>1.000000000000000</double>
    </property>
    <property name="upperBound">
     <double>4095.000000000000000</double>
    </property>
    <property name="scaleMaxMajor">
     <number>5</number>
    </property>
    <property name="scaleMaxMinor">
     <number>3</number>
    </property>
    <property name="scaleStepSize">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QwtThermo" name="potenciometro">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>91</width>
      <height>261</height>
     </rect>
    </property>
    <property name="upperBound">
     <double>4095.000000000000000</double>
    </property>
   </widget>
   <widget class="QwtTextLabel" name="text_alarma_adc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>LEDS</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue</string>
    </property>
    <property name="text">
     <string>Enciende</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado Cliente APP: Detenido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FRECUENCIA EJECUCIÃ“N</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>runButton</zorder>
  <zorder>label</zorder>
  <zorder>leHost</zorder>
  <zorder>pingButton</zorder>
  <zorder>groupBox_25</zorder>
  <zorder>board_status</zorder>
  <zorder>TextLabel_3</zorder>
  <zorder>statusLabel</zorder>
  <zorder>label_15</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ManoMeter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/manometer.h</header>
  </customwidget>
  <customwidget>
   <class>ThermoMeter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/thermometer.h</header>
  </customwidget>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
